ssg/controls.py

```diff
@@ -0,0 +1,104 @@
+import collections
+import logging
+import os
+
+import ssg.yaml
+import ssg.utils
+
+class Control():
+    def __init__(self):
+        self.id = None
+        self.rules = []
+        self.variables = {}
+
+    @classmethod
+    def from_control_dict(cls, control_dict):
+        control = cls()
+        control.title = control_dict.get("title")
+        control.description = control_dict.get("description")
+        control.automated = control_dict.get("automated", False)
```

Comment:
This assumes that automated is set to "no" by default. I suggest ensuring that "yes", "no" and "partially" are the only acceptable inputs, and we consider "yes" by default, so users can use the `automated` key in cases when it has a chance to "surprise" people.